lib/browser/chrome-extension.js

```diff
@@ -240,7 +240,7 @@ const getMessagesPath = (extensionId) => {
 
 ipcMainUtils.handleSync('CHROME_GET_MESSAGES', async function (event, extensionId) {
   const messagesPath = getMessagesPath(extensionId)
-  return fs.promises.readFile(messagesPath)
+  return fs.promises.readFile(messagesPath).then(b => b.toString('utf8'))
```

Comment:
Can we not send buffers over IPC anymore??

Reply:
you totally can, but there's no particular reason to here, and the remote end will end up as a Uint8Array, which doesn't have this form of `toString()`.